frontend/src/features/SideWindow/MissionForm/utils.ts

```diff
@@ -0,0 +1,68 @@
+import { omit } from 'ramda'
+
+import { Mission, MissionData, MissionSource } from '../../../domain/types/mission'
+import { FormError } from '../../../libs/FormError'
+import { hasMissingOrUndefinedValues } from '../../../utils/hasMissingOrUndefinedValues'
+
+import type { MissionFormValues } from './MainForm/types'
+
+/**
+ * Are `<missionFormValues>` complete enough to be transformed into a `MissionData` type and sent to the API?
+ */
+export function isCompleteMissionFormValues(missionFormValues: MissionFormValues | undefined): boolean {
+  try {
+    if (!missionFormValues) {
+      return false
+    }
+
+    getMissionDataFromMissionFormValues(missionFormValues)
+
+    return true
+  } catch (_) {
+    return false
+  }
+}
+
+export function isValidControlUnit(
+  formResourceUnit: MissionFormValues['controlUnits'][0]
+): formResourceUnit is Mission['controlUnits'][0] {
+  return !hasMissingOrUndefinedValues(['administration', 'contact', 'id', 'name', 'resources'], formResourceUnit)
```

Comment:
the `contact` field is nullable

Reply:
But it is written as mandatory there: https://github.com/MTES-MCT/monitorenv/blob/loup/use-identifiers-with-units/backend/src/main/kotlin/fr/gouv/cacem/monitorenv/domain/entities/controlResources/ControlUnitEntity.kt#L6

Reply:
It's not : https://github.com/MTES-MCT/monitorenv/blob/loup/use-identifiers-with-units/backend/src/main/kotlin/fr/gouv/cacem/monitorenv/domain/entities/controlResources/ControlUnitEntity.kt#L8